<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<arg name="num_of_agents" default="10"/>
	<arg name="folder_loc" default="/home/kota/ego_swarm_data/csv/"/>
	<arg name="sim" default="00"/>

	<node name="ave_distance" pkg="ego_planner" type="ave_distance.py" output="screen">
		<param name="num_of_agents" value="$(arg num_of_agents)"/>
		<param name="folder_loc" value="$(arg folder_loc)"/>
		<param name="sim" value="$(arg sim)"/>
	</node>

</launch>
